test/test_proxy_tensor.py

```diff
@@ -1566,8 +1567,6 @@ def f(a, b, c, d, e):
     xfail('float_power', ''),  # base given to float_power_ has dtype Float but the operation's result requires dtype Double
     # decomp not implemented
     xfail('unique', ''),
-    # in-place has a different signature than out-of-place
-    xfail('uniform', ''),
```

Comment:
Previously, we were setting the random seed for trace_f(*args) but not setting it for f(*args). This PR changes it so that we set the random seed for both. Because random operators are run in the same order, we can expect the output of both of these to be the same.

Reply:
oof, nice catch